pkg/config/config.go

```diff
@@ -385,11 +386,64 @@ func InitCliConfig(configAndStacksInfo schema.ConfigAndStacksInfo, processStacks
 			atmosConfig.StackType = "Logical"
 		}
 	}
+	setLogConfig(&atmosConfig)
 
 	atmosConfig.Initialized = true
 	return atmosConfig, nil
 }
 
+func setLogConfig(atmosConfig *schema.AtmosConfiguration) {
+	// TODO: This is a quick patch to mitigate the issue we can look for better code later
```

Comment:
```suggestion
	// TODO: This is a quick patch to mitigate the issue we can look for better code later
```

Please elaborate on "the issue" that it's a quick fix for so that someone reading this would understand, without previous context.

Reply:
Added issue link :)